highfidelity/hifi
# Change default mouse speeds for avatar

Test plan
- Open Settings->Avatar->Avatar Settings
- Check that Camera Pitch Speed equals 45 and Camera Yaw Speed equals 60
- Check mouse look around speed in Desktop mode while right click and drag. It should be slower then in previous build
